draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -435,33 +477,52 @@
         <t>It is out of the scope of this document how the preference index is
         determined or configured.</t>
 
-        <t>The Preference Index Sub-TLV has the following format:</t>
+        <t>The Site Preference Index Sub-TLV has the following format:</t>
 
-        <figure anchor="Preference Index Sub-TLV"
-                title="Preference Index Sub-TLV">
+        <figure anchor="Site Preference Index Sub-TLV"
+                title="Site Preference Index Sub-TLV">
           <artwork><![CDATA[ 
    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
   |Site-Preference-Index Sub-Type | Length        | Reserved      |
   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                   Preference Index value                      |
+  |                Site Preference Index value                    |
   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ 
 	]]></artwork>
         </figure>
+	<!-- XXX JMH - The normative text below should indicate the field
+	lengths.  PDU diagrams disagreeing with the normative text is
+	unfortunately common in IETF and can lead to interop issues.
+
+	If the sub-tlv format is described above as discussed in a prior
+	comment, there is no need to re-state the sub-type and length sizes -
+	but also no harm in doing so. -->
 
         <t><list style="hanging">
             <t hangText="-">Site-Preference-Index Sub-Type =1 (specified in
             this document).</t>
+	    <!-- XXX JMH - style comment: This is a sub-tlv of value 1, which is
+	    assigned for site-preference index.  It is not a a site-preference
+	    index sub-tlv.  Consider how you would like to name such things.
+	    -->
```

Comment:
Not clear of your comment. 
The Type for the Site-Preference_index Sub-TLV is assigned as 1. How to state it more accurately?

Reply:
What I'm saying is the sub-tlv format is:
2-octets sub-tlv type.
1-octet tlv length

In a specific instance of a sub-tlv, you're setting the code point for the sub-tlv type; e.g. 1.  You're not renaming each instance of the type field to be a new sub-tlv definition.

Reply:
how about using the following format to describe the Sub-TLV Type across the document? 

"Site-Preference-Index Sub-Type (16 bits): 1 (specified in this document)"

Reply:
This is consistent in the document, so better.  Mostly I expect that the format ends up being something like "sub-tlv 1: thing name" rather than "sub-tlv thing name: value 1".

I suspect the RFC editor will be the tie breaker here, but since this is now consistent, we can resolve this item.